src/plugins/custom_integrations/common/index.ts

```diff
@@ -51,6 +51,13 @@ export const INTEGRATION_CATEGORY_DISPLAY = {
   geo: 'Geo',
 };
 
+// featured integrations will be brought to the top of the search results for
+// a given category. Integrations are displayed in the order of the array.
+export const FEATURED_INTEGRATIONS_BY_CATEGORY = {
+  security: ['endpoint', 'cloud_security_posture', 'network_traffic'],
+  '': ['apm', 'endpoint', 'web_crawler'], // no category selected
```

Comment:
Using `''` as a key here is a bit interesting. I suppose it makes sense, butI wonder if it'd be a bit clearer to use a string like `'none'` to provide the default list?

Reply:
I was a bit uncomfortable about '' as a key, the thing that swayed it for me is that it is type guarded by the function signature of promoteFeaturedIntegrations, e.g '' is a valid category but undefined or none aren't